### ISSUE TITLE
App not working. 

### ISSUE THREAD

**Issue by 013Harkey** [g-mocken/ZoeStatus#12]
I've entered the new key (YjkKtHmGfaceeuExUDKGxrLZGGvtVS0J) but still the app says VIN index not found, I noticed there are 5 index's in settings could this be causing problems. 
![Uploading IMG_0418.png…]()

**Comment by g-mocken** [comment]
If you have only _one_ car registered with the MyR service, then you must leave VIN index at the _first_ position. If you then see a different error message, post it here.

**Comment by 013Harkey** [comment]
Sent from my iPhoneOn 20 Apr 2023, at 9:59 pm, Guido Mocken ***@***.***> wrote:﻿
If you have only one car registered with the MyR service, then you must leave VIN index at the first position. If you then see a different error message, post it here.

**Comment by ZeelandSeaView** [comment]
App not working anymore... Can you fix this please,it's the only app for
My Zoe with a Apple Watch app.

**Comment by durosity** [comment]
> App not working anymore... Can you fix this please, it's the only app for my Zoe with an Apple Watch app.
> 
> Thanks!
> 
> Zoe R110 52kw

I don’t think this is being maintained anymore.  You could try the Zeddy app which is similar.  Annoying the chap who maintains that no longer has a Renault himself, but he’s still issuing updates to fix issues, at least as of a few days ago.

(apologies if I’ve assumed this is no longer being updated incorrectly)

**Comment by ZeelandSeaView** [comment]
Found the latest api here:

https://github.com/hacf-fr/renault-api/blob/main/src/renault_api/const.py#L21


YjkKtHmGfaceeuExUDKGxrLZGGvtVS0J

And it works!

**Comment by g-mocken** [comment]
> I don’t think this is being maintained anymore. You could try the Zeddy app which is similar. Annoying the chap who maintains that no longer has a Renault himself, but he’s still issuing updates to fix issues, at least as of a few days ago.
> 
> (apologies if I’ve assumed this is no longer being updated incorrectly)

You are right that there has not been any update in quite some time, but the main reason is that there was no reason for it (and some minor new features are simply not ready for release, yet). There were some issues with the backend several weeks ago, but that affected also Zeddy and the original Renault app in the same way - and it was resolved apparently by Renault after some time.

I still own a ZOE and for me everything is correctly working, even without changing the API key again.

**Comment by Macrisu** [comment]
Well, is not working again.
I think Renault changed something.

**Comment by ZeelandSeaView** [comment]
Same here, again not working. Is there another option to see my Zoe status on Apple Watch?

**Comment by g-mocken** [comment]
Confirmed ... apparently the URL https://renault-wrd-prod-1-euw1-myrapp-one.s3-eu-west-1.amazonaws.com/configuration/android/config_de_DE.json is no longer available (403). I spent all day trying to figure that out, because most of my devices were still loading the JSON file from some old local cache (without ever requesting it over the network at all) and, as a result, they made ZOE status appear to be working perfectly. 
I now have a preliminary code that seems to work again, but it will take a couple of days, before this will be in the store again.

**Comment by ZeelandSeaView** [comment]
Thanks for the good work!

**Comment by g-mocken** [comment]
You can try it out using [TestFlight](https://testflight.apple.com/join/lT0KBqzh).

**Comment by Macrisu** [comment]
The error in testflight is that you are not accepting any testers now

**Comment by g-mocken** [comment]
I know, but the error message is misleading ... it's actually in App Store review.

**Comment by g-mocken** [comment]
Now it's active.

**Comment by ZeelandSeaView** [comment]
The app works well!

**Comment by SimonFromRegensburg** [comment]
Is the watch app still supported? It is not working any more (at least for me). E.g.,  no force touch functionality to enter the menu, no credentials stored, no data retrieval.

Would be great to update the watch app as well - as it is still kind of unique!

**Comment by g-mocken** [comment]
Yes, and it is still working on my Series 3 with watchOS 8. 

I do however see compiler warnings 
` WKInterfaceController Menu Items are deprecated since watchOS 7.0
`
which are clearly related to the long-press activated menu items, so there is a chance that Apple has silently dropped support for this. What version of watchOS are you running?

The good news is, that I have anyway started rewriting certain parts of the app a couple of weeks ago, and also been working on the watch app for about a week, and getting rid of those deprecation warnings was one of my goals. Currently, my development version has a simple button for a three-item menu in place of the long-press activated hidden menu items.
If you are a developer, you can compile it yourself from the "swift-watch-extension" branch, otherwise you will have to wait a little while, until I find the time to do a new TestFlight (see link in the Readme) or even AppStore release.

**Comment by SimonFromRegensburg** [comment]
Hi Guido,Thanks for feedback so quickly after my post.Meanwhile I do have an Apple Watch 9, running on watchOS 11.3.1. I think the broken functionally started with my watch update quite some time ago.It’s great to hear that you are still maintaining the watchOS version of Zoe Status. Especially the watchOS version, because there is no alternative to that on the App Store. Another great thing of your watch app is, that it was working on the cellular watch without the iPhone connected. Many apps have that limitation. I guess Apple is very restrictive here.Looking forward to your next update. Let me know if can provide further data to support you! (Gerne auch auf deutsch :-) Viele Grüße,Simon Schwalenberg Am 02.04.25 um 18:36 schrieb Guido Mocken
                
            
            
                
                    Von: "Guido Mocken" ***@***.***>Datum: 2. April 2025An: "g-mocken/ZoeStatus" ***@***.***>Cc: "Comment" ***@***.***>,"SimonFromRegensburg" ***@***.***>Betreff: Re: [g-mocken/ZoeStatus] App not working.  (Issue #12)
                
                
  Yes, and it is still working on my Series 3 with watchOS 8.
I do however see compiler warnings
WKInterfaceController Menu Items are deprecated since watchOS 7.0
which are clearly related to the long-press activated menu items, so there is a chance that Apple has silently dropped support for this. What version of watchOS are you running?
The good news is, that I have anyway started rewriting certain parts of the app a couple of weeks ago, and also been working on the watch app for about a week, and getting rid of those deprecation warnings was one of my goals. Currently, my development version has a simple button for a three-item menu in place of the long-press activated hidden menu items.
If you are a developer, you can compile it yourself from the "swift-watch-extension" branch, otherwise you will have to wait a little while, until I find the time to do a new TestFlight (see link in the Readme) or even AppStore release.
—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>




g-mocken left a comment (g-mocken/ZoeStatus#12)

Yes, and it is still working on my Series 3 with watchOS 8.
I do however see compiler warnings
WKInterfaceController Menu Items are deprecated since watchOS 7.0
which are clearly related to the long-press activated menu items, so there is a chance that Apple has silently dropped support for this. What version of watchOS are you running?
The good news is, that I have anyway started rewriting certain parts of the app a couple of weeks ago, and also been working on the watch app for about a week, and getting rid of those deprecation warnings was one of my goals. Currently, my development version has a simple button for a three-item menu in place of the long-press activated hidden menu items.
If you are a developer, you can compile it yourself from the "swift-watch-extension" branch, otherwise you will have to wait a little while, until I find the time to do a new TestFlight (see link in the Readme) or even AppStore release.

**Comment by g-mocken** [comment]
I have submitted a new version to TestFlight.

**Comment by g-mocken** [comment]
Note that there is a minor bug in the transfer of credentials. Will be fixed soon.

**Comment by g-mocken** [comment]
Should be fixed in TestFlight v1.4.0.1.
